packages/markdown-support/package.json

```diff
@@ -19,19 +19,19 @@
   "dependencies": {
     "@silvenon/remark-smartypants": "^1.0.0",
     "github-slugger": "^1.3.0",
-    "gray-matter": "^4.0.2",
-    "mdast-util-mdx-expression": "^1.0.0",
+    "gray-matter": "^4.0.3",
+    "mdast-util-mdx-expression": "^1.1.0",
     "micromark-extension-mdx-expression": "^1.0.0",
-    "rehype-raw": "^5.1.0",
-    "rehype-stringify": "^8.0.0",
-    "remark-footnotes": "^3.0.0",
-    "remark-gfm": "^1.0.0",
-    "remark-parse": "^9.0.0",
-    "remark-rehype": "^8.1.0",
-    "remark-slug": "^6.1.0",
-    "unified": "^9.2.1",
+    "rehype-raw": "^6.0.0",
+    "rehype-stringify": "^9.0.1",
+    "remark-footnotes": "^4.0.1",
+    "remark-gfm": "^2.0.0",
+    "remark-parse": "^10.0.0",
+    "remark-rehype": "^9.0.0",
+    "remark-slug": "^7.0.0",
+    "unified": "^10.1.0",
     "unist-util-map": "^3.0.0",
-    "unist-util-visit": "^3.1.0"
+    "unist-util-visit": "^4.0.0"
```

Comment:
Types change, as far as I could tell it didn't affect us: https://github.com/syntax-tree/unist-util-visit/releases/tag/4.0.0